ringbuffer/ringbuffer.c

```diff
@@ -80,7 +81,7 @@ ssize_t ringbuf_get_memsize(const unsigned count)
         return -EINVAL;
 
     ssize_t sz = sizeof(ringbuf_t) + count * sizeof(void *);
```

Comment:
The right way is to mention the size, which is multiple of cache line size.

Reply:
It will be alright to limit the size to the multiple of cache line size. But in this situation, I think `ALIGN_FLOOR` won't change anything of the original `sz`. Is there any reason to use `ALIGN_FLOOR` here?

Reply:
@RinHizakura, you are right. `ALIGN_CEIL` should be used for allocating dynamic memory. I was thinking of the memory pool which manages the elements where `ALIGN_FLOOR` was used.